docs/userGuide/syntax/siteNavigationMenus.mbdf

```diff
@@ -3,26 +3,26 @@
 **A _Site Navigation Menu_ (==_siteNav_ for short==) is a fixed menu on the ==left edge== of a page**, that can be used to show a road map of the main pages of your site.
 
 Steps to add a siteNav:
-1. Format your siteNav as an unordered Markdown list and save it inside the `_markbind/navigation` directory.
+1. Format your siteNav as an unordered Markdown list and save it inside the `_markbind/navigation` directory. You may include dividers in the siteNav via <a href="{{ baseUrl }}/userGuide/readerfacingfeatures#horizontal-rules">horizontal rules</a>.
 2. Specify the file as the value of the `siteNav` attribute in the `<frontmatter>` of the page.
 
 <div class="indented">
 
 {{ icon_example }} Here is an example siteNav code saved in **`_markbind/navigation/`**`mySiteNav.md` file:
 
-<box>
-
-<code>
-* [:house: Home]({{ showBaseUrlText }}/index.html)<br>
-* Docs<br>
-&nbsp;&nbsp;* [User Guide]({{ showBaseUrlText }}/ug.html)<br>
-&nbsp;&nbsp;* [Dev Guide]({{ showBaseUrlText }}/dg.html)<br>
-* [Search]({{ showBaseUrlText }}/search.html)<br>
-&nbsp;&nbsp;* [Google Search](https://www.google.com/)<br>
-&nbsp;&nbsp;* [YouTube Search](https://www.youtube.com/)<br>
-* [Contact]({{ showBaseUrlText }}/contact.html)
-</code>
-</box>
+```{.no-line-numbers}
+* [:house: Home]({{ baseUrl }}/index.html)
+---
+* Docs
+  * [User Guide]({{ baseUrl }}/ug.html)
+  * [Dev Guide]({{ baseUrl }}/dg.html)
+---  
```

Comment:
Bit of a nit, but let's not leave trailing spaces behind 🙂 

You should also be able to configure your editor to automatically remove trailing whitespace behind, so you don't have to worry about it yourself.